examples/api-routes-graphql/package.json

```diff
@@ -6,12 +6,14 @@
     "start": "next start"
   },
   "dependencies": {
-    "apollo-server-micro": "^3.0.1",
-    "graphql": "^15.5.1",
-    "micro": "^9.3.4",
+    "@graphql-yoga/node": "^2.2.1",
+    "graphql": "^16.3.0",
     "next": "latest",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "swr": "^0.5.6"
+  },
+  "resolutions": {
+    "graphql": "16.3.0"
```

Comment:
Is this resolution needed? Ideally we don't need to include this as it won't apply for `npm`

Reply:
@ijjk Just tested without the resolutions and pushed a commit to remove it